network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/profile_directory.html

```diff
@@ -0,0 +1,11 @@
+{% load static %}
+
+<div class="profile-directory">
+  <div class="profile-filters">
+    {% for label in filters %}
+    <button class="filter-button">{{ label }}</button>
+    {% endfor %}
+  </div>
+
+  {% include "./profile_blocks.html" %}
```

Comment:
even if the JS comes in a different PR, this PR should probably have the changes to `profile_blocks` that adds a `data-year`, `data-profile-type`, and `data-program-type` that the filtering will be tapping into.